### DISCUSSION TITLE
如果可以，让韩国服务器滚粗吧

### DISCUSSION THREAD

**Discussion by Psydean** [getlantern/lantern-forum-cn#988] [General]
跳韩日之后根本就打不开，一直转圈，好难啊！ 7.9.9。一加 12

**Comment by MaxMKMax** [comment]
@Psydean 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解

**Reply by Psydean** [reply]
老板，真的，点击发送又说技术问题。
这种跳韩国服务器然后宕机的现象，让蓝灯变得好难用啊
